DBSetup/createDB.sql

```diff
@@ -1,31 +1,40 @@
 create database VA_Trial;
 \c va_trial
+
+create table Patient (
+PatientID int NOT NULL,
+Deceased bool NOT NULL,
+FirstName text NOT NULL, 
+LastName text NOT NULL,
+Email text NOT NULL,
```

Comment:
FirstName, LastName are for sure personally identifiable information, Email might be personally identifiable information. Thought we might not actually insert anything into these, HIPPA regulations are spooky and I'm afraid of ghost.
Are we able to remove these or they **have** to be in database. And if they have to be, I think we'll need to open a dialog about storing PII.

Reply:
When we did the presentation for Candice she mentioned that she wanted Firstname, Lastname, and email to be included in the patients table. I don't think this will violate HIPPA at all. She seemed pretty sure of it.